crowbar_framework/app/models/node_object.rb

```diff
@@ -256,6 +256,28 @@ def intended_role=(value)
     @node["crowbar_wall"]["intended_role"] = value
   end
 
+  def disks_roles=(value)
+    @node["crowbar_wall"] ||= {}
+    @node["crowbar_wall"]["disks_roles"] = value
+  end
+
+  def disks_roles
+
+    disks_roles = crowbar_wall["disks_roles"] || {}
+
+    # Check for current unclaimed disks, and add unclaimed disk to
+    # disks_roles hash if it is not there yet.
+    # (New disk could have been added to the node)
+    unclaimed_physical_drives.each do |d, data|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `data`. If it's necessary, use `_` or `_data` as an argument name to indicate that it won't be used.